xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1398,7 +1398,14 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
-  if (fabs(m_fps - g_graphicsContext.GetFPS()) < 0.01)
+  // refresh rate can be a multiple of video fps
+  double diff;
+  if (g_graphicsContext.GetFPS() >= m_fps)
+    diff = fmod(g_graphicsContext.GetFPS(), m_fps);
```

Comment:
This doesn't look quite right. Suppose:
``g_graphicsContext.GetFPS() = 59.94``  ``m_fps=30``
``diff = .998`` so clocksync is not enabled, but I think it should be.